content/how-to/tutorial-create-equipment-class-rule.md

```diff
@@ -0,0 +1,355 @@
+caption="A new unit of measure created in the UI."
+>}}
+
+### Creating a BPMN workflow
+
+A rule must trigger a BPMN workflow.
+Before setting up a rule, create its workflow.
+For this example, this simple 3-node BPMN is enough:
+
+[rules_example_bpmn](../../static/BPMNs/rules_example_bpmn.bpmn)
+
+1. Navigate to **Workflows > Process List**.
+2. Import the BPMN.
+3. Save it.
+4. Set the version as active.
+
+{{< bigFigure
+width="100%"
+alt="A BPMN created in the UI."
+src="/images/equipment-class-rules/screenshot-rhize-Creating_a_BPMN.png"
+caption="A BPMN created in the UI."
+>}}
+
+The BPMN has a `Libre Jsonata Transform` task that contains an expression `"Preheating oven for order number " & $.orderNumber"` .
+The rule engine triggers this BPMN with a payload that includes the order number value, as follows:
+
+```json
+{
+  "orderNumber": "Order1"
+}
+```
+
+### Create an equipment class with bound properties
+
+#### Equipment class and version
+
+1. Navigate to **Master Data > Equipment Class**.
+2. Create a new equipment class from the sidebar. The label might be `Pizza Line`, for example.
+3. From the `General` tab, create a new Draft version.
+
+{{< bigFigure
+width="100%"
+alt="A new equipment class and version created in the UI."
+src="/images/equipment-class-rules/screenshot-rhize-Creating_an_Equipment_Class_and_Version.png"
+caption="A new equipment class and version created in the UI."
+>}}
+
+#### Equipment class property
+
+1. From the properties tab, create a new property.
+2. For type, select `BOUND`.
+3. For name, enter `orderNumber`.
+4. For UoM, select the unit of measure created earlier (`Order Number`).
+
+{{< bigFigure
+width="100%"
+alt="A new equipment class property created in the UI."
+src="/images/equipment-class-rules/screenshot-rhize-Creating_an_Equipment_Class_Property.png"
+caption="A new equipment class property created in the UI."
+>}}
+
+## Create a rule
+
+### Add a rule to an existing equipment class
+
+1. From the Rules tab of an equipment class version, create a new rule.
+2. Enter `Run BPMN on Order Number` for the name and confirm.
+3. Select `rules_example_bpmn` for the workflow specification.
+4. Add `orderNumber` as a trigger property.
+5. Add a trigger expression that evaluates to true or false.
+
+{{% notice note %}}
+
+The rule runs the preceding workflow only if the expression evaluates to `true`.
+It’s common to compare the new value with the previous.
+
+{{% /notice %}}
+
+In this case, we can compare the new order number to the previous by adding `orderNumber.current.value != orderNumber.previous.value`.
+Note that the root of the object path must match the ID of the equipment class property we set up earlier.
+If the property were called `Order Number`, you would access it like this instead `"Order Number".current.value != "Order Number".previous.value`.
+
+The entire information that becomes available to the rule engine looks like this:
+
+```javascript
```

Comment:
Hey @DevDimov , just wondering, how did you get this JSON? Where did it come from? It might make sense to add the Query or whatever in so users can reproduce their own.